Lib/posixpath.py

```diff
@@ -572,7 +570,7 @@ def commonpath(paths):
                 common = s1[:i]
                 break
 
-        prefix = sep if isabs else sep[:0]
+        prefix = sep if paths[0].startswith(sep) else sep[:0]
```

Comment:
Can `sep[:0]` ever be anything other than `''`?

Reply:
It could be `b''`. But it indeed looks a bit odd

Reply:
Ah yes got it thanks.